settings_vial/settings_vial.py

```diff
@@ -37,15 +37,17 @@ def __repr__(self):
 
     def __getattr__(self, attr):
         if self.override_prefix and self.override_keys_function:
-            override_set = self._load_override_set()
+            override_keys = self._load_override_keys()
 
-            if override_set - set(self._override_config):
+            if set(override_keys) - set(self._override_config):
                 warnings.warn(
-                    "There's no configuration with override keys {}".format(override_set - set(self._override_config)),
+                    "There's no configuration with override keys {}".format(
+                        set(override_keys) - set(self._override_config)
+                    ),
                     MissingOverrideKeysWarning,
                 )
 
-            for key in override_set:
+            for key in reversed(override_keys):
```

Comment:
I'd prefer to make this behavior at least dependent on another instance var of `Settings`, something like: `override_keys_reverse_lookup=[False|True]`. And then, only reverse if this boolean is true.

Reply:
That's much better indeed. I updated the pr to add a `override_keys_reverse_lookup` kwarg to control this behaviour.